MQTT_server.md

```diff
@@ -0,0 +1,38 @@
+# Introduction
+The structure of MQTT server consists of three components. `Raspberry Pi client`, `TA server` and `Parser`
```

Comment:
You don't have to address Raspberry Pi while talking about the structure. It can be referred to any functioned endpoint. For some integrated solutions like Qualcomm SoC, it is straightforward to be at the same board.

Reply:
I will remove all the word `raspberry pi` in the document, since as you siad I should write a document to descide a more general solution.

Reply:
I will use `NB-IoT client` to replace `Raspberry pi client`